### PR TITLE
Lp preparation for handover

### PR DESCRIPTION
**Pull request by LoganDSPrice**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,43 +1,35 @@
...
-%w(Residential Institutional Mixed-Use Hospitality Commercial).each do |c|
-  Category.create(name:c)
-end
-
 user_data = [
-  {username:"Logan", email:"l@l.com", password: "password"},
-  {username:"Pavan", email:"p@p.com", password: "password"},
-  {username:"Arjun", email:"a@a.com", password: "password"},
-  {username:"Forefront", email:"f@f.com", password: "password"}
+  {username:"Forefront", email:"f@f.com", password: "password"},
```
Comment (pksarguru) [inline db/seeds.rb]: we should tell them to change this email and password when they first log in.

Comment (LoganDSPrice) [inline db/seeds.rb]: Oh, yeah. Maybe the best choice would be to remove all User seeds so they create the first user the first time they go to the admin page. That way if for some reason they have to reset/reseed they don't have to remember the username/password combo. What do you think?

Comment (pksarguru) [inline db/seeds.rb]: yea that's best.  We should do what we need to do and then wipe all users.

Comment (LoganDSPrice) [inline db/seeds.rb]: That would, however mean that we'd need them to make us accounts for us to be able to do anything on it.

Comment (pksarguru) [inline db/seeds.rb]: I mean we can access the database and make ourselves accounts, right?

File: vendor/extensions/articles/db/seeds.rb [hunk 2]
```diff
@@ -20,31 +20,33 @@
 
 Refinery::Articles::Article.destroy_all
 
-Refinery::Articles::Article.create(
+if Rails.env.downcase != "production"
```
Comment (pksarguru) [inline vendor/extensions/articles/db/seeds.rb]: seems odd we need to use `.downcase` here

File: db/seeds.rb (hunk 1 above)
Comment (LoganDSPrice) [inline db/seeds.rb]: oh yeah! that's true.

File: vendor/extensions/articles/db/seeds.rb (hunk 2 above)
Comment (LoganDSPrice) [inline vendor/extensions/articles/db/seeds.rb]: we probably don't. I was just being conservative since we're comparing a string.

Comment (pksarguru) [inline vendor/extensions/articles/db/seeds.rb]: http://www.rubydoc.info/github/rails/rails/Rails.env

yea it looks like you don't need it

Comment (LoganDSPrice) [inline vendor/extensions/articles/db/seeds.rb]: i was probably inconsistent with it though, yeah?

Comment (pksarguru) [inline vendor/extensions/articles/db/seeds.rb]: it's in a couple places but if you search of `.env.downcase` that's probably going to be thorough enough

**Pull request merged**